wp_api/src/lib.rs

```diff
@@ -218,3 +206,31 @@ macro_rules! generate {
 }
 
 uniffi::setup_scaffolding!();
+
+#[cfg(test)]
+mod lib_tests {
```

Comment:
When making this mod as `tests` like we usually do in other source files, running this mod in VS code would run the tests in the whole library. Maybe the "tests" mod at the root level is reserved by cargo or something.

Reply:
I run all the commands from the command line, so I am not 100% sure, but I think this is the expected behavior. I assume VS code triggers the tests for the mod you're running and since this is the root mod, everything else falls under it. If my assumption is correct, than running tests in `request` module should also run all the tests under `endpoint` module. I unfortunately can't test this right now because I don't have VS code setup.

Reply:
Also, I think I'd prefer if we used the same name for the mod for all unit tests. It makes it easier to run them from the command line as we don't need to remember what the mod name is. It may also effect some partial matches.

```suggestion
mod tests {
```

Reply:
So, I took a look what VS code does. Below is the command VS code runs when click "Run tests" button under a `mod test_mod_name`:

```
cargo test --package wp_api --lib --all-features -- <test_mod_name> --show-output 
```

When naming this module as `tests`, VS code would run `cargo test [...] -- tests`. Since pretty much every test function would have `tests` in their qualified name, all tests are executed.